### DISCUSSION TITLE
Audacious install

### DISCUSSION THREAD

**Discussion by jump01** [audacious-media-player/audacious#1434] [General]
I want to install Audacious on my Macbook but I don't know how to do it. can you help me?

**Comment by radioactiveman** [comment]
On macOS you can use Homebrew to install Audacious.

https://audacious-media-player.org/download

**Reply by sanchezelton** [reply]
I want to see if I can make a Homebrew cask so it gets an icon I can manage directly from the Dock, but [instructions for doing this](https://docs.brew.sh/Adding-Software-to-Homebrew#creating-the-cask-file) require a *.dmg or *.zip file. The formula in Homebrew provides a bottle, but neither nor ZIP or DMG.

**Reply by ghost** [reply]
once you installed it with home brew, you can use the Automator (from apple default tools) and create an app from "run shell script".
